apps/native-component-list/src/screens/DateTimePickerScreen.tsx

```diff
@@ -1,96 +1,119 @@
-import React from 'react';
-import moment from 'moment';
+import {
+  SafeAreaView,
+  ScrollView,
+  StyleSheet,
+  View,
+  Text,
+  StatusBar,
+  Button,
+  Platform,
+} from 'react-native';
 import DateTimePicker from '@react-native-community/datetimepicker';
-import { StyleSheet, View, Text, Button, Platform } from 'react-native';
+import React, { useState } from 'react';
+import moment from 'moment';
 
 // This example is a copy from https://github.com/react-native-community/react-native-datetimepicker/tree/master/example
 // Please try to keep it up to date when updating @react-native-community/datetimepicker package :)
 
-type DateTimeMode = 'date' | 'datetime' | 'time';
+const DateTimePickerScreen = () => {
+  const [date, setDate] = useState(new Date(1598051730000));
```

Comment:
is it your birthday?

Reply:
Hah, I wish to be born on 21st August 2020 😉 
This example is mostly just a copy from:
https://github.com/react-native-community/react-native-datetimepicker/blob/e6d2980230b1e27fa8aed989ccbb79897e6ef5eb/example/App.js#L16-L18